src/styles/spinner.ts

```diff
@@ -0,0 +1,30 @@
+import styled, { keyframes } from 'styled-components';
+import COLORS from './colors';
+
+const spinKeyframes = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+// $size - controls width and height
+// $primary - color of the spinning arc
+// $secondary - color of the full circle background
+interface SpinnerProps {
+  $size?: string;
+  $primary?: string;
+  $secondary?: string;
+}
+
+export const Spinner = styled.div<SpinnerProps>`
+  display: inline-block;
+  border-radius: 99999px;
```

Comment:
uh LMFAO wouldn't this be 50%?

Reply:
no, `50%` makes it an ellipse, whereas `99999px` makes this a pill
ref: https://stackoverflow.com/questions/29966499/border-radius-in-percentage-and-pixels-px-or-em